### DISCUSSION TITLE
Add stack-trace to the selector-memo warning?

### DISCUSSION THREAD

**Discussion by infinite-persistence** [reduxjs/react-redux#2065] [General]
I really like these warnings, but I don't known which selector the warning is referring to because anonymous functions are being used in the repo I'm working on.

<img width="633" alt="image" src="https://github.com/reduxjs/redux-toolkit/assets/64950861/b832042e-92af-4d83-9dc8-4c66c8b9da83">

If the stack trace is added (in addition to the existing `selected|selected2|state`), that would point me to the right direction.  

Thoughts?  Or perhaps I'm missing something here?

**Comment by markerikson** [comment]
Seems like something we could add, yeah.

**Comment by EskiMojo14** [comment]
opened https://github.com/reduxjs/react-redux/pull/2064 - that has a build that it'd be great if you could try 🙂
